pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -77,7 +79,7 @@ func NewGKENetworkParamSetController(
 		gkeNetworkParamsInformer: gkeNetworkParamsInformer,
 		networkInformer:          networkInformer,
 		gceCloud:                 gceCloud,
-		queue:                    workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "gkenetworkparamset"),
+		queue:                    workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), workqueueName),
```

Comment:
use NewRateLimitingQueueWithConfig instead (NewNamedRateLimitingQueue is deprecated), see https://github.com/kubernetes/cloud-provider-gcp/blob/master/pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go#L109C20-L109C20

Reply:
done